pkgs/tools/graphics/ldgallery/viewer/node-composition.nix

```diff
@@ -0,0 +1,17 @@
+# This file has been generated by node2nix 1.8.0. Do not edit!
```

Comment:
For some reason, npm now fails to find some dependency.

```
npm ERR! code ENOTCACHEDtchPackageMetaData error for eslint-K
npm ERR! request to https://registry.npmjs.org/eslint-utils failed: cache mode is 'only-if-cached' but no cached response available.

npm ERR! A complete log of this run can be found in:
npm ERR!     /build/.npm/_logs/2020-05-02T15_46_11_921Z-debug.log

builder for '/nix/store/wkac3zzw316cx7mzh70ikw7j9ammcbl7-node_ldgallery-viewer-1.0.0.drv' failed with exit code 1
cannot build derivation '/nix/store/fqjisvda6pvd9lpj5rbapsqiwfslmliy-ldgallery-viewer.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/6whc0iz2qgzmv59nh7nlycv2l1h0ma33-ldgallery-1.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/6whc0iz2qgzmv59nh7nlycv2l1h0ma33-ldgallery-1.0.drv' failed
```

"eslint-utils" is however present (twice) in the generated ./node-packages.nix.

Reply:
This problem seems to have disappeared when packaging the v2.0 instead of v1.0.